src/Raw2Gltf.cpp

```diff
@@ -570,43 +629,53 @@ ModelData *Raw2Gltf(
 
         /** Create a new TextureData for the given RawTexture index, or return a previously created one. */
         auto getSimpleTexture = [&](int rawTexIndex, const std::string &tag) {
-            const std::string key = texIndicesKey({ rawTexIndex }, tag);
+            const std::string key = texIndicesKey({rawTexIndex}, tag);
             auto iter = textureByIndicesKey.find(key);
-            if (iter != textureByIndicesKey.end()) {
+            if (iter != textureByIndicesKey.end())
+            {
                 return iter->second;
             }
 
-            const RawTexture  &rawTexture      = raw.GetTexture(rawTexIndex);
-            const std::string textureName      = StringUtils::GetFileBaseString(rawTexture.name);
+            const RawTexture &rawTexture = raw.GetTexture(rawTexIndex);
+            const std::string textureName = StringUtils::GetFileBaseString(rawTexture.name);
             const std::string relativeFilename = StringUtils::GetFileNameString(rawTexture.fileLocation);
 
             ImageData *image = nullptr;
-            if (options.outputBinary) {
+            if (options.outputBinary)
+            {
                 auto bufferView = gltf->AddBufferViewForFile(buffer, rawTexture.fileLocation);
-                if (bufferView) {
+                if (bufferView)
+                {
                     std::string suffix = StringUtils::GetFileSuffixString(rawTexture.fileLocation);
                     image = new ImageData(relativeFilename, *bufferView, suffixToMimeType(suffix));
                 }
-
-            } else if (!relativeFilename.empty()) {
+            }
+            else if (!relativeFilename.empty())
+            {
                 image = new ImageData(relativeFilename, relativeFilename);
-                std::string outputPath = outputFolder + relativeFilename;
-                if (FileUtils::CopyFile(rawTexture.fileLocation, outputPath)) {
-                    if (verboseOutput) {
+
```

Comment:
Alright, I scanned all the changes in this file, and I **think** this tweak is the only thing that's not just whitespace. I'll manually merge this in a bit, working on that assumption.